packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -731,9 +731,9 @@ curl -v -X POST \
 | -------- | ------------------------------------------------------------------------------ | ---------- | ---------- | -------- |
 | type     | The authentication scheme type. Currently the only supported type is `HEADER`. | String     | TRUE       | FALSE    |
 | key      | The header name for the authorization header                                  | String     | TRUE       | FALSE    |
-| value    | The header value                                                              | String     | TRUE       | TRUE     |
+| value    | The header value. This is the secret value that you want Okta to pass to your external service endpoint (so that your external service can check for its presence as a security measure).                                                               | String     | TRUE       | TRUE     |
 
-To use Basic Auth, set `type` to `HEADER`, `key` to `Authorization`, and `value` to the Base64-encoded string of "username:password".
+To use Basic Auth, set `type` to `HEADER`, `key` to `Authorization`, and `value` to the Base64-encoded string of "username:password". Make sure to include the scheme as part of the `value` parameter. For example, `Basic YWRtaW46c3VwZXJzZWNyZXQ=`. See [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) for further information.
```

Comment:
> Ensure that you include the authentication scheme as part of the `value` parameter. For example, `Basic YWRtaW46c3VwZXJzZWNyZXQ=`. See [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication)~ for further information~.

(changed to style guidelines for "ensure" vs. "make sure" and for hyperlink text).